src/browser/modules/Stream/SysInfoFrame/SysInfoFrame.tsx

```diff
@@ -0,0 +1,205 @@
+/*
```

Comment:
Could we perhaps use git commands to rename this file? As it stands now, it looks like we'll lose the commit history on this file after it was renamed...

Reply:
I was thinking a single commit that only renamed would be enough, I'll try git mv!

Reply:
Seems to me from this stack overflow thread it doesn't matter if I use git mv instead? https://stackoverflow.com/questions/2314652/is-it-possible-to-move-rename-files-in-git-and-maintain-their-history